test/unit/components/navbars.spec.js

```diff
@@ -21,15 +21,15 @@ describe('components', function() {
 
     describe('navbarAbsoluteTop', function() {
       it('adds class has-navbar-top if navbarAbsoluteTop is present', function() {
-        let elem = angular.element('<div class="navbar-absolute-top" />');
+        let elem = angular.element('<div class="navbar-absolute-top" nav-controller />');
```

Comment:
That's a nice trick :D

That kind of coverage is useless, coverage is just a number after all, and unit tests requires isolation to be reliable. 

Try to write a test suite for your feature that works independently from the rest.

Or leave it as it was and i'll try to cover it.